Documentation/guides/vscode-support.md

```diff
@@ -0,0 +1,86 @@
+# VSCode Support
+
+Xamarin.Android itself can be developed within VSCode. There is
+a workspace included in the repo `Xamarin.Android.code-workspace`.
+The required extensions should be installed when you open the
+workspace.
+
+## Building Xamarin.Android
+
+Follow the instructions [here]() to build Xamarin.Android on the
+command line first. Then open the `Xamarin.Android.code-workspace`
+in vscode.
+
+## Running/Debugging Unit Tests
+
+Xamarin.Android uses the `nxunit-explorer` extension to allow you to run and
+debug unit tests. This extension should be installed automatically.
+In order to run or debug the tests you will need to set a path
+to the `nunit-console.exe` file. This is will be located in your
+global nuget cache, usually  in `$HOME/.nuget`.
+
+The setting you need to set is `nxunitExplorer.nunit`. This will need
+to be done in your `Preferences->Settings` in VSCode. This will save
+the setting globally so it will be available when ever you open
+Xamarin.Android.
+
+Alternatively you can add something like this to the `Xamarin.Android.code-workspace`.
+
+```json
+"nxunitExplorer.nunit": "/Users/<user>/.nuget/packages/nunit.consolerunner/3.11.1/tools/nunit3-console.exe"
+```
+
+The path will change depending on where your nuget cache is and depending on which
+Nunit version is being used.
+
+Once that is setup you can to go the `Testing` tab in VSCode and browse/run/debug
+the unit tests.
+
+Note: There is a slight problem with the nxunitExplorer in that it does not escape
+the names of the unit tests it runs. As a result the unit tests which contain
+arguments do not run within VSCode. However a work around is to run the top
+level fixture rather than the single test.
+
+Alternatively you can download the patched executable from [here](https://github.com/dellis1972/vscode-nxunit-test-adapter/releases/tag/Patch2) and replace
+your `testrun` files with these. This will allow you to run and debug all the
+tests.
+
+### Running the Sample
+
+Xamarin.Android provides a few Debug configurations in VSCode to debug the
+sample applications `HelloWorld` and `VSAndroidApp`. You can select the
+`Debug Sample` Run and Debug config to run and debug the samples under
+`mono` or you can use `Debug Sample Under DotNet` to debug this under the
+.net 6 system.
+
+In order to use the `Debug Sample Under DotNet` you will need to have built
+Xamarin.Android for .net 6. See the windows [documentation](../building/windows/instructions.md) or
```

Comment:
I think you should "expand" the link text, having `[windows documentation](…)` and `[\*nix documentation](…)`.